tests/cpp/test_resize.cpp

```diff
@@ -1900,6 +1902,11 @@ TEST_F(ResizeTest, FusionSliceForNanoGPT2) {
 
 // C++ version of TestNvFuserFrontend.test_nanogpt_split_mha_linears
 TEST_F(ResizeTest, FusionSliceForNanoGPT3) {
+  // To verify input caching condition in this test, disable aliasing as that
+  // will skip compilation and no kernel will exist.
+  preseg_passes::OptimizationPassGuard<preseg_passes::MarkAliasesPreparePass>
+      optimization_guard(false);
+
   auto fusion_ptr = std::make_unique<Fusion>();
```

Comment:
I know you didn't add this, but since you are here, do you mind doing a minor, side refactor? 

1. Change this line to
   ```
   auto fusion = std::make_unique<Fusion>();
   ```
2. Remove the next line says `auto& fusion=*fusion_ptr`. 
3. Change existing uses of `fusion.` to `fusion->`. 

The gist is that `unique_ptr` comes with syntax sugar to make it as if it's a raw pointer. It's not necessary to create another variable just to hold its reference.